frontend/src/Services/PollService.ts

```diff
@@ -0,0 +1,110 @@
+import AuthAPI from '@api/AuthAPI'
+import { PollAPI } from '@api/PollAPI'
+
+import { Poll, PollBackendResponse, PollVoteRequest, PollVoteResponse } from '../Types/Poll'
+import { BatchedCache } from '../Utils'
+
+class PollService {
```

Comment:
`PollService` doesn't match the pattern of other APIs. It's better placed with other APIs/API helpers